lib/datadog/core/environment/ext.rb

```diff
@@ -40,6 +40,8 @@ module Ext
         TAG_SERVICE = 'service'
         TAG_VERSION = 'version'
 
+        DD_SITE_STAGING = 'datad0g.com'
+
```

Comment:
I left a suggestion below to avoid this (and my reasoning). Even if we do keep the code as-is, I don't think we should have this constant here -- we should embed it in the code that uses it, as only that bit of code should ever know about this info.